style.css

```diff
@@ -492,9 +496,200 @@ form {
   font-size: 17px;
 }
 
+.space-date {
+  margin-bottom: 15px;
+}
+
+.modal-3 {
+  list-style: none;
+  width: 100%;
+  font-family: "Poppins", sans-serif !important;
+  background-color: #fff;
+  color: #0011a8;
+  border: 1px solid #0011a8;
+  border-radius: 8px;
+  padding: 16px 32px;
+  font-size: 15px;
+  text-align: center;
+}
+
+.modal-2 {
+  list-style: none;
+  background-color: #e5e5e5;
+  color: #6070ff;
+  border: 1px solid #e5e5e5;
+  border-radius: 8px;
+  padding: 8px 12px;
+  font-size: 15px;
+}
+
+.modal-1 {
+  color: #6070ff;
+  display: flex;
+  gap: 4px;
+  padding-bottom: 20px;
+  margin-top: 20px;
+  font-size: 12px;
+}
+
+.column-left-1 p {
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 15px;
+  color: #344563;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  margin: 12px 0;
+}
+
+.pop-close {
+  display: block;
+  position: absolute;
+  top: 20px;
+  right: 20px;
+  color: #6b778c;
+}
+
+.card-portfolio-1 {
+  background-color: #fff;
+  border-radius: 8px;
+  box-shadow: 0 0 5px rgba(0, 0, 0, 0.3);
+  width: 343px;
+  height: 630px;
+  padding: 24px;
+  margin: 24px auto;
+  display: none;
+  position: fixed;
+  top: 0;
+  bottom: 0;
+  right: 0;
+  left: 0;
+  overflow: hidden;
```

Comment:
- [ ] Please change your overflow from `overflow: hidden;` to `overflow: y;` to enable vertical scroll, especially for the mobile version. Currently, I can't scroll to see extra details.
![image](https://user-images.githubusercontent.com/45460192/189437718-afc9a7d8-ba7c-4bb3-9e4f-7f745adb8d16.png)